console/console-init/ui/src/components/JsonViewEditor/JsonViewEditor.tsx

```diff
@@ -3,22 +3,75 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import React from "react";
+import React, { useState } from "react";
 import { Editor } from "components";
+import {
+  PageSection,
+  PageSectionVariants,
+  Button,
+  Tooltip,
+  TooltipPosition,
+  ButtonVariant
+} from "@patternfly/react-core";
+import { CopyIcon } from "@patternfly/react-icons";
+
 interface IJsonViewEditorProps {
   detailInJson?: any;
   readOnly: boolean;
+  style?: React.CSSProperties;
 }
+
 const JsonViewEditor: React.FunctionComponent<IJsonViewEditorProps> = ({
   detailInJson,
```

Comment:
Can we revisit the naming here, my suggestion is deviceDetailInJson.

Reply:
Will update this in my next PR